main.py

```diff
@@ -120,7 +120,7 @@ def main():
 
     ui_cb = ui.ui_callbacks()
     global_vars.freeze_button.configure(command=ui_cb.freeze_unfreeze)
-    update_interval_slider_label.configure(text=f"Update interval: \
+    update_interval_slider_label.configure(text=f"Update Response interval: \
```

Comment:
After this change the text for the label is not completely visible in the UI

Reply:
Good catch!